slack_sdk/socket_mode/async_client.py

```diff
@@ -122,6 +122,8 @@ async def process_messages(self):
             while not self.closed:
                 try:
                     await self.process_message()
+                except asyncio.CancelledError:
+                    raise
```

Comment:
@vectoralpha Perhaps, changing this part as below would be safer. What do you think?
```suggestion
                except asyncio.CancelledError:
                    # if self.closed is True, the connection is already closed
                    # In this case, we can ignore the exception here
                    if not self.closed:
                        raise
```

Reply:
Let us go with this change